internal/common/convert_types.go

```diff
@@ -237,9 +237,18 @@ func ExpandStringList(configured []interface{}) []*string {
 	return vs
 }
 
+// value is nil. set null.
 func Int64ValueFromInt32(value *int32) basetypes.Int64Value {
 	if value == nil {
 		return basetypes.NewInt64Null()
 	}
 	return basetypes.NewInt64Value(int64(*value))
 }
+
+// value is nil. set 0.
```

Comment:
could you add more context as to why we need to set a zero value instead of null here in the comment (with FIXME)? and, it seems to be better to have named, for example, `Int64FromInt32OrDefault()` (trivial)